math/src/dynamic_fees/math.rs

```diff
@@ -1,91 +1,139 @@
 use super::types::*;
 use crate::dynamic_fees::types::NetVolumeDirection::{InOut, OutIn};
+use num_traits::One;
 use num_traits::Zero;
 use sp_arithmetic::traits::Saturating;
+use sp_arithmetic::FixedPointNumber;
 use sp_arithmetic::{FixedPointOperand, FixedU128, PerThing};
 
 /// Recalculate Omnipool's asset fee given previously calculated fee and oracle data.
 ///
 /// `volume` is the asset volume data provided by the oracle.
```

Comment:
this can be removed as it is not a param anymore

Reply:
no it cannot be - it is trait needed for some operations